packages/commercetools/helpers/src/index.ts

```diff
@@ -27,3 +31,15 @@ export const getCategoryProducts = (category: Category, options: any = {}): Prod
 
   return _products
 }
+
+export const getProductVariants = (products: ProductVariant[], options: any = {}): ProductVariant | ProductVariant[]  => {
```

Comment:
I've seen a couple of times that option master true. Maybe it's worth to add an interface for it?

Reply:
that's right, but there are different logic, for finding master one